addons/web/static/src/views/form/form_controller.scss

```diff
@@ -174,7 +174,7 @@ $o-form-label-margin-right: 0px;
     }
 
     // No sheet
-    .o_form_nosheet {
+    &.o_form_nosheet, .o_form_nosheet {
```

Comment:
Is the second part still useful? I don't think so

Reply:
I'm not sure, see https://github.com/odoo/odoo/pull/98072#discussion_r946822576